src/process.cr

```diff
@@ -479,17 +474,7 @@ class Process
     argv << Pointer(UInt8).null
 
     LibC.execvp(command, argv)
-
-    error_message = String.build do |io|
-      io << "execvp ("
-      command.inspect_unquoted(io)
-      args.try &.each do |arg|
-        io << ' '
-        arg.inspect(io)
-      end
-      io << ")"
-    end
-    raise Errno.new(error_message)
+    raise RuntimeError.from_errno
```

Comment:
Why is this not `from_error("execvp")`?

Reply:
I think eventually we should make an effort to improve many error messages. I'm not sure it's a good idea to leak the syscall name that failed. Maybe the same error message we have above: "Error executing process"

Reply:
Yes, that could work. "execvp" has always just been a stand-in for a proper message.

Using an explicit message here would make `from_errno` without arguments obsolete. I think it should be required to provide at least some message for context information.

Reply:
Is it leaking the syscall name or telling you exactly what failed, which manpage to read and what to search for?

Reply:
I actually don't think leaking the name is an issue. It's really pretty concise about defining the context of the error. (Btw. it's a libc call, not a syscall)

But in other cases, I think we can perhaps be more expressive about the error context. There's a huge problem space with localization, though. So IDK, but using just the call looks like a pretty reasonable solution, at least for the time being.